authentication.js

```diff
@@ -49,7 +49,8 @@ const refreshAccessToken = (z, bundle) => {
 
     const result = JSON.parse(response.content);
     return {
-      access_token: result.access_token
+      access_token: result.access_token,
+      refresh_token: result.refresh_token
```

Comment:
@bcooksey not sure what would happen if an API would not have a `refresh_token`. The object this JS would then return `refresh_token:undefined`. Any idea how we'd receive that in Python and if that would unset an existing `refresh_token` when we do https://github.com/zapier/zapier/blob/6ea9aa6c5fca319a6522ffe5992138d9f08efbf1/developer_cli/api_base.py#L717?

In that case this would need to be:

```js
const auth_fields = {
  access_token: result.access_token
}
if (result.refresh_token) {
  auth_fields.refresh_token = result.refresh_token
}
return auth_fields
```

Or `_.pick(result, 'access_token', 'refresh_token')`

Reply:
You are correct that it would overwrite the existing refresh token with an empty one, so being defensive here is probably a good idea.

Thanks for the PR @johnpan!

Reply:
my pleasure! @bcooksey

Reply:
@johnpan For us to accept, mind signing the CLA. There should be a link in the merge checks for this PR

Reply:
@johnpan also, could you make the code compatible with situations that return no refresh token to not overwrite any existing? See my code example above.